interfaces/apparmor/template.go

```diff
@@ -336,6 +336,12 @@ var templateCommon = `
   owner @{HOME}/snap/@{SNAP_INSTANCE_NAME}/                  r,
   owner @{HOME}/snap/@{SNAP_INSTANCE_NAME}/**                mrkix,
 
+  # Experimental snap folder changes
```

Comment:
nitpick: this comment is going to become obsolete when we merge this PR, isn't it? :-)